packages/shared/src/components/post/PostComments.tsx

```diff
@@ -116,6 +112,23 @@ export function PostComments({
   const { hash: commentHash } = window.location;
   const commentsCount = comments?.postComments?.edges?.length || 0;
   const commentRef = useRef(null);
+  const deleteCommentPrompt = async (
+    commentId: string,
+    parentId: string | null,
+  ) => {
+    const options: PromptOptions = {
+      title: 'Delete comment',
+      description:
+        'Are you sure you want to delete your comment? This action cannot be undone.',
+      okButton: {
+        title: 'Delete',
+        className: 'btn-primary-ketchup',
+      },
+    };
+    if (await showPrompt(options)) {
+      await deleteComment({ commentId, parentId, post });
+    }
```

Comment:
Would it not be neater to attach this to the actually button?
The cancel doesn't go to this resolve?

Reply:
this is not a callback. `showPrompt` returns `true` for okButton, and `false` for the cancel.